src/components/ProgressionButton/ProgressionButton.js

```diff
@@ -2,25 +2,30 @@ import './ProgressionButton.scss';
 import Forward from "./assets/arrow-forward.png"
 import Backward from "./assets/arrow-back.png";
 import React from "react";
+import { Link } from "react-router-dom";
```

Comment:
These changes defeat the purpose of what this component is for - you can create a new component for navigation but leave this as it is, it's meant for button clicks that lead to something executing